### PR TITLE
Fix tab navigation order in issue creation popup

### LINKED ISSUE #4493: Tab navigation order issue in Issue Creation popup
**Description:** When creating a new issue in the project's issue tracker, there is an inconsistency in the tab navigation order within the issue creation popup. After setting the Assignee, pressing the Tab key should move the focus to the next logical field, which is expected to be 'Labels'. However, the focus jumps to 'Estimation', and subsequently, it moves to 'No component'.

**Steps to Reproduce:**

1. Open the issue creation modal by clicking on 'New issue'.
2. Fill in the 'Issue title' and 'Description' fields.
3. Use the Tab key to navigate to the 'Assignee' field and set an assignee.
4. Press Tab to move to the next field.

**Expected Behavior:** The focus should move to the 'Labels' field after 'Assignee' when navigating through the fields using the Tab key.

**Actual Behavior:** The focus jumps to the 'Estimation' field, skipping the 'Labels' field, and then moves out of sequence to the 'No component' field.

navigation1

navigation2

### PR DESCRIPTION
**Pull request by ilaydasimsek**
Fixes https://github.com/hcengineering/platform/issues/4493. When creating a new issue in the project's issue tracker, the Tab navigation was working out of order and not jumping to the next field. To adress this bug:

* Fixed the focus order of ComponentSelector and EstimationEditor
* TagsDropdownEditor was ignored when using Tab navigation, added the missing focusIndex to TagsDropdownEditor to address the issue

Now the tab button click traverses the components in correct order:

https://github.com/hcengineering/platform/assets/36786266/169d36cc-6ee6-4d11-bcf7-aec3fbcc3f21

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjBjMjgwZmU1MGZjYjI5NThiMDVlNmUiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.QG29JqxBma2WQVeGp2z3lLo-9BkIgYprJpj0wO4s1kg">Huly&reg;: <b>UBERF-6288</b></a></sub>

### CODE REVIEW

**Comment by aonnikov** [comment]
Looks good. What about due date and parent issue, is that something that is easy to fix?

**Comment by ilaydasimsek** [comment]
@aonnikov Thanks! It should be pretty easy to fix, I wasn't sure if due date and parent were left out intentionally. I could make a fix for those as well if that's not the case

**Comment by aonnikov** [comment]
I'm ok with making them available via tab navigation

**Comment by ilaydasimsek** [comment]
@aonnikov added due date and parent to tab navigation as well

https://github.com/hcengineering/platform/assets/36786266/5816a839-c6cd-449b-b4df-406cd112660b

**Comment by aonnikov** [comment]
@ilaydasimsek  can you please sign-off commits?

**Comment by ilaydasimsek** [comment]
@aonnikov Sure, done now

File: packages/ui/src/components/calendar/DatePresenter.svelte [hunk 1]
```diff
@@ -26,11 +26,13 @@
   import DPCalendar from './icons/DPCalendar.svelte'
   import DPCalendarOver from './icons/DPCalendarOver.svelte'
   import { getMonthName } from './internal/DateUtils'
+  import { registerFocus } from '../../focus'
 
   export let value: number | null | undefined
   export let mode: DateRangeMode = DateRangeMode.DATE
   export let mondayStart: boolean = true
   export let editable: boolean = false
+  export let input: HTMLButtonElement | undefined = undefined
```
Comment (aonnikov) [inline packages/ui/src/components/calendar/DatePresenter.svelte]: should it be exported? do you expect it to be passed from outside of the component?

Comment (ilaydasimsek) [reply packages/ui/src/components/calendar/DatePresenter.svelte]: That is right, I don't expect it to be passed from outside, removed the export on [712ef3f](https://github.com/hcengineering/platform/pull/5148/commits/712ef3f59da3fb303fbf01a20aa0451475f003a2)

**Review by haiodo — APPROVED** [review]

**Comment by haiodo** [comment]
Hi @ilaydasimsek thanks for a contribution.

**Pull request merged**